packages/nx/src/command-line/yargs-utils/shared-options.ts

```diff
@@ -192,6 +193,7 @@ export function withDepGraphOptions(yargs: Argv): Argv {
       describe:
         'Output file (e.g. --file=output.json or --file=dep-graph.html)',
       type: 'string',
+      alias: ['out'],
```

Comment:
remove for now....

cc @philipjfulcher how do you think someone should be able to run `nx graph` so that it outputs to the terminal?

We are making `nx graph --file stdout` print to the terminal... but it looks kind of weird.. so our preliminary thought was `nx graph --out stdout` looks better.. but we don't have to make this identical to `file`. This could just be like `--stdout` or `--print`